packages/node_modules/pouchdb-replication/src/replicate.js

```diff
@@ -149,6 +150,7 @@ function replicate(src, target, opts, returnValue, result) {
   function getDiffs() {
     var diff = {};
     currentBatch.changes.forEach(function (change) {
+      returnValue.emit('checkpoint', { 'diffing': change });
```

Comment:
slight preference for calling this `revs_diff` as that’s what the CouchDB API for this is called.